modules/artifact/src/main/java/org/mule/runtime/module/artifact/api/descriptor/ClassLoaderModel.java

```diff
@@ -157,10 +182,34 @@ public ClassLoaderModelBuilder exportingResources(Set<String> resources) {
       return this;
     }
 
+    /**
+     * Indicates which Java packages are loaded from the artifact itself, even if some other artifact in the region exports it.
```

Comment:
I would clarify why you can ignore the packages already provided on the region. As discussed the other day, the reason is because the API exported on the artifact does not contain theses packages, but they are contained on the artifact's jars.